### PR TITLE
changing to the correct version

### PR DESCRIPTION
**Pull request by DiegoAntonino**

### CODE REVIEW

**Comment by daddykotex** [comment]
👍  good to me

how can publish to npm?

**Comment by lmcjerl** [comment]
@daddykotex I can publish, do you want do know who, or how?

File: nodes/mnubo/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name"         : "node-red-contrib-mnubo",
-    "version"      : "1.2.0",
+    "version"      : "1.3.0",
     "description"  : "Mnubo",
     "dependencies": {
         "mnubo-sdk": "^1.7.0",
```
Comment (lmcjerl) [inline nodes/mnubo/package.json]: Do you need version 1.9.0?

Comment (DiegoAntonino) [inline nodes/mnubo/package.json]: using '^' will grab the latest 1.x.x version of 'mnubo-sdk'.

Comment (DiegoAntonino) [inline nodes/mnubo/package.json]: done!

**Pull request merged**